decidim-proposals/app/controllers/concerns/decidim/proposals/orderable.rb

```diff
@@ -56,6 +56,8 @@ def reorder(proposals)
             proposals.order(proposal_votes_count: :desc)
           when "recent"
             proposals.order(created_at: :desc)
+          else
+            proposals
```

Comment:
I agree with you but if we remove this we need to ensure that the `detect_order` function has a default value in the case the `order` is not found, what do you think?